lib/bolt/cli.rb

```diff
@@ -244,7 +244,7 @@ def handle_parser_errors
 
     def puppetdb_client
       return @puppetdb_client if @puppetdb_client
-      puppetdb_config = Bolt::PuppetDB::Config.load_config(nil, config.puppetdb)
+      puppetdb_config = Bolt::PuppetDB::Config.load_config(nil, config.puppetdb, config.boltdir)
```

Comment:
We check for `$future` in PuppetDB::Config, so this won't change behavior if you don't have `future` set.